deploy/README.md

```diff
@@ -81,8 +81,8 @@ The following table displays the tested Kubernetes and Helm versions.
 | K8s with GKE  | 1.20<br/>1.21<br/>1.22          |
 | K8s with AKS  | 1.21<br/>1.22                   |
 | OpenShift     | 4.6<br/>4.7<br/>4.8             |
-| Helm          | 3.5.4 (Linux)                   |
-| kubectl       | 1.16.0                          |
+| Helm          | 3.8.2 (Linux)                   |
```

Comment:
I suggest to also change the Helm version in `Minimum Requirements` section above to 3.5+ due to this issue: https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/2261